2-patterns_integrations/src/main/java/com/epam/dhontar/aqamp/utils/enums/ServicesEndpoints.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.dhontar.aqamp.utils.enums;
+
+public enum ServicesEndpoints {
+    //    SITE_URL("https://fakerestapi.azurewebsites.net/index.html"),
+    BASE_URL("https://fakerestapi.azurewebsites.net/api/v1/"),
+    AUTHORS_URL("Authors/"),
+    USERS_URL("Users/"),
+    SLACK_API_URL("");
+
+    private final String urlValue;
+
+    ServicesEndpoints(String urlValue) {
+        this.urlValue = urlValue;
+    }
+
+    public String getValue() {
+        return urlValue;
+    }
+}
```

Comment:
Clean this class please. Remove unused veriables

Reply:
did you mean that I have to remove the commented string that I left for myself to not forget the correct site URL for the test API?